ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/management-gnmi.yml

```diff
@@ -1,6 +1,8 @@
 management_api_gnmi:
+  notification_timestamp:
+    send_time: true
   enable_vrfs:
     MGMT:
       access_group: ACL-GNMI
     MONITORING:
-  octa:
+  octa: true
```

Comment:
This is a data model change, and cannot be accepted in a minor release.
Unfortunately the `management_api_gnmi.octa` key is also valid with a `none` value. It was done like that originally, but should be fixed in the next major release.
Please revert this and all associated changes in the templates. Ex. you have to use `is defined` since it also passes on `none` value.

Reply:
There is a problem with this. The eos template is using the ```arista.avd.defined``` test while the documentation is using ```defined``` test. So this is a bug which doesn't match with the README as well.